site/develop/index.md

```diff
@@ -115,4 +123,3 @@ We value all contributions that help to build a vibrant community, not just code
 You can contribute by testing the code, helping verify a release,
 writing documentation or the web site,
 or just by answering questions on the list.
-
```

Comment:
Was it intended?

Reply:
Does not appear in regular git diff, I don't know why it appears here.

Reply:
Well, it does appear in the console `git diff` for me (macos). Original file ended up with two `0x0a` characters and a fixed one has just one.

On the other hand, let's just fix it.

Reply:
It's my editor who ate the line :P I add it back!